nextflow.config

```diff
@@ -24,6 +24,7 @@ params {
     normalize                   = true
     compute_freq                = false
     remove_samples              = null
+    chunk_model                 = 'sequential'
```

Comment:
There is no test assessing that the sequential mode works.

Reply:
For small regions it seems that the 'sequential' mode create more than two overlapping regions at the start of the chromosome, however it is the recommended model for large region.

Reply:
Fair enough! and we test this with the full sized test right? So there is no need to add a small test here as it is at least covered by the full size test.

Reply:
I agree !
I've tried it on my HPC and it worked well with this parameter.